plugins/nf-nomad/src/main/nextflow/nomad/NomadConfig.groovy

```diff
@@ -44,19 +44,31 @@ class NomadConfig {
     }
 
     class NomadClientOpts{
+        private Map<String,String> sysEnv
```

Comment:
I think we don't need to have a reference to sysEnv only use it in the constructor